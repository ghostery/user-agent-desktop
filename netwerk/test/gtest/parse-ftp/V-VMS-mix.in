This is a composite of 'difficult' VMS (MultiNet/UCX/CMU) LIST lines.
CAUTION WHILE EDITING - lines may have dangling CRs! (intentional)


Directory DISK$VOL:[DIR1.DIR2]
DISK$VOL:
ACA:[ANONYMOUS]

AUTOEXEC.BAT;1      %RMS-E-PRV, insufficient privilege or file protection violation
README.FTP;1        insufficient privilege or file protection violation
LOGIN.COM;2                 1   4-NOV-1994 14:09 [ANONYMOUS] (RWE,RWE,,)
CII-MANUAL.TEX;1      213/216  29-JAN-1996 03:33:12  [ANONYMOU,ANONYMOUS] (RWED,RWED,,)
THIS-IS-A-LONG-VMS-FILENAME.WITH-CR-TO-NEXT-LINE                      213/216  29-JAN-1996 03:33:12  [ANONYMOU,ANONYMOUS] (RWED,RWED,,)
ANOTHER-LONG-VMS-FILENAME.WITH-LF-TO-NEXT-LINE
                          213  29-JAN-1996 03:33 [ANONYMOU,ANONYMOUS] (RWED,RWED,,)
[A.DIR]CMU-VMS-IP-FTP-FILE;1 1/3 5-MAR-1993 18:09
MAX_FILESIZE.FILE;1   2199023255040/4294967295 5-MAR-1993 18:09

Total of 710 blocks in 12 files.
226 Transfer completed.
ftp> quit
221 QUIT command received. Goodbye.
