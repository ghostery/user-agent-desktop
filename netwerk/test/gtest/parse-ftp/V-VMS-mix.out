<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

11-04-1994  14:09:00                  512 LOGIN.COM
01-29-1996  03:33:12               109056 CII-MANUAL.TEX
01-29-1996  03:33:12               109056 THIS-IS-A-LONG-VMS-FILENAME.WITH-CR-TO-NEXT-LINE
01-29-1996  03:33:00               109056 ANOTHER-LONG-VMS-FILENAME.WITH-LF-TO-NEXT-LINE
03-05-1993  18:09:00                  512 CMU-VMS-IP-FTP-FILE
03-05-1993  18:09:00        2199023255040 MAX_FILESIZE.FILE
